src/services/recipient.js

```diff
@@ -523,13 +527,13 @@ export async function getGoalsByActivityRecipient(
       'source',
       'goalTemplateId',
       [sequelize.literal(`
-        CASE 
-          WHEN COALESCE("Goal"."status",'')  = '' OR "Goal"."status" = 'Needs Status' THEN 1 
-          WHEN "Goal"."status" = 'Draft' THEN 2 
-          WHEN "Goal"."status" = 'Not Started' THEN 3 
-          WHEN "Goal"."status" = 'In Progress' THEN 4 
-          WHEN "Goal"."status" = 'Closed' THEN 5 
-          WHEN "Goal"."status" = 'Suspended' THEN 6 
+        CASE
+          WHEN COALESCE("Goal"."status",'')  = '' OR "Goal"."status" = 'Needs Status' THEN 1
```

Comment:
Should we be using ENUM (constants) here?

Reply:
Perhaps, but this particular change is just the removal of some unnecessary trailing whitespace.